themes/default/content/docs/get-started/b.md

```diff
@@ -0,0 +1,511 @@
+- **Self-Managed**: a manually managed object store, including AWS S3, Azure Blob Storage, Google Cloud Storage, any AWS S3 compatible server such as Minio or Ceph, or your local filesystem
+
+{{% chooser backend "service,self-managed" / %}}
+
+{{% choosable backend service %}}
+
+To create a Pulumi Cloud account run the `login` command:
+
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ pulumi login</code></pre>
+</div>
+
+And that's it! The Pulumi Cloud backend requires no additional configuration after [installing the CLI](/docs/install/) and logging in. Pulumi offers this backend hosted online free for individuals, with [advanced tiers](/pricing/) available for teams and enterprises (with <a href="https://app.pulumi.com/site/trial" target="_blank">free trials</a>).
+
+> To learn more about the Pulumi Cloud backend's design, including why it doesn't need your cloud credentials, see [Pulumi Cloud Architecture](#/pulumi-cloud/-architecture).
+
+{{< /choosable >}}
+
+{{% choosable backend self-managed %}}
+
+If you decide to use a self-managed backend, you will need to be more aware of how state works (see [Advanced State](/docs/concepts/state/#advanced-state)). If you lose the checkpoint for your stack, or it drifts from reality, Pulumi will not behave as you might expect &mdash; for instance, if your state file is empty, Pulumi will think your stack is empty, and will attempt to recreate all of the resources. Some commands may also behave slightly differently between backends. For example, the Pulumi Cloud ensures there are no other updates in flight for a given stack, and in general, reliability, security, and collaboration is automatic with the Pulumi Cloud.
+
+If you already have a self-managed backend set up you can login via the `login` command:
+
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash">$ pulumi login $BACKEND_URL</code></pre>
+</div>
+
+If you do not have a self-managed backend set up, please refer to [Using a Self-Managed Backend](/docs/concepts/state/#using-a-self-managed-backend) documentation for information on how to set one up.
+
+{{< /choosable >}}
+
+## Install Language Runtime
+
+Select your language:
+
+{{< chooser language "javascript,typescript,python,go,csharp,java,yaml" / >}}
+
+{{% choosable language "javascript,typescript" %}}
+{{< install-node >}}
+{{% /choosable %}}
+
+{{% choosable language python %}}
+{{< install-python >}}
+{{% /choosable %}}
+
+{{% choosable language go %}}
+{{< install-go >}}
+{{% /choosable %}}
+
+{{% choosable language "csharp,fsharp,visualbasic" %}}
+{{< install-dotnet >}}
+{{% /choosable %}}
+
+{{% choosable language "java" %}}
+{{< install-java >}}
+{{% /choosable %}}
+
+{{% choosable language "yaml" %}}
+{{< install-yaml >}}
+{{% /choosable %}}
+
+## Pick a Starter Project
+
+Pick from the starter projects below to start creating, modifying, and destroying infrastructure with Pulumi:
```

Comment:
Nits, but "choose" sounds a teeny bit nicer than "pick", IMO, and I'd suggest removing "modifying and destroying", since the user won't "start" by doing either of those. 

```suggestion
## Choose a starter project

Choose from the list of starter projects below to start creating infrastructure with Pulumi:
```

Reply:
👍